visualization/app/codeCharta/codeCharta.service.ts

```diff
@@ -41,7 +45,10 @@ export class CodeChartaService {
 			}
 		})
```

Comment:
Completely change independent comment: @NearW using a for loop her would allow to break the loop instead of having to use the `printErrors` variable.

Reply:
I KNEW THERE WAS DIFFERENCE BETWEEN FOR AND FOREACH. AAAAH.
Thanks.